src/TransactionalSession.MsSqlSystemDataClient.AcceptenceTests/TransactionalSession.MsSqlSystemDataClient.AcceptenceTests.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\NServiceBusTests.snk</AssemblyOriginatorKeyFile>
+    <LangVersion>10.0</LangVersion>
+    <!-- We want the root namespace to match the transactional session one -->
+    <RootNamespace>NServiceBus.TransactionalSession.AcceptanceTests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\TestHelper\TestHelper.csproj" />
+    <ProjectReference Include="..\NServiceBus.Persistence.Sql.TransactionalSession\NServiceBus.Persistence.Sql.TransactionalSession.csproj" />
+    <ProjectReference Include="..\SqlPersistence\SqlPersistence.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+    <PackageReference Include="Nunit" Version="3.13.3" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0-rc.1" />
+    <PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="7.0.0-beta.2" />
+  </ItemGroup>
+
+  <Import Project="..\ScriptBuilderTask\NServiceBus.Persistence.Sql.targets" />
+
+  <ItemGroup>
+    <Compile Include="..\TransactionalSession.MsSqlMicrosoftDataClient.AcceptanceTests\Infrastructure\CaptureServiceProviderStartupTask.cs">
```

Comment:
Simplify to Infrastructure\*.cs ?

Reply:
It's not all files I'm including though...

Reply:
> It's not all files I'm including though...

should the folder structure then be adjusted to allow easier sharing of common files?

Reply:
I've done that already